schema/model/mixins/spin_orbit_coupling.json

```diff
@@ -0,0 +1,10 @@
+{
+    "schemaId": "model/mixins/spin-orbit-coupling",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+        "spinOrbitCoupling": {
+            "type": "boolean"
```

Comment:
"title": "Spin-orbit coupling inclusion flag"

Reply:
Per our discussion, we will use a title manifest in `science_meve`.